### PR TITLE
Update the list of example desktop agents

### PR DESCRIPTION
**Pull request by ggeorgievx**

### CODE REVIEW

File: website/versioned_docs/version-1.2/api/spec.md [hunk 1]
```diff
@@ -11,10 +11,9 @@ A Desktop Agent is a desktop component (or aggregate of components) that serves
 
 Examples of Desktop Agents include:
 
-- Autobahn
 - Cosaic's Finsemble
+- Glue42
 - OpenFin
-- Refinitiv Eikon
```
Comment (kriswest) [inline website/versioned_docs/version-1.2/api/spec.md:17]: Didn't Refinitiv talk about work they were doing on FDC3 at a meeting a couple of months ago? I would suggest leaving this in.

**Review by kriswest — CHANGES_REQUESTED** [review]

File: website/versioned_docs/version-1.2/api/spec.md (hunk 1 above)
Comment (rikoe) [reply website/versioned_docs/version-1.2/api/spec.md:17]: @ggeorgievx @kriswest same with Autobahn actually - let's not use this change to remove others, let's just add Glue42.

**Comment by ggeorgievx** [comment]
@kriswest @rikoe I applied the requested changes and squashed the commits for a cleaner merge. Thank you!

**Review by kriswest — APPROVED** [review]
LGTM

**Pull request merged**